src/http.cc

```diff
@@ -1926,9 +1926,9 @@ HttpStateData::httpBuildRequestHeader(HttpRequest * request,
         delete cc;
     }
 
-    /* maybe append Connection: keep-alive */
-    if (flags.keepalive) {
-        hdr_out->putStr(Http::HdrType::CONNECTION, "keep-alive");
+    /// RFC7230 section 6.6
```

Comment:
```suggestion
    // HTTP/1.1 connections are persistent by default (RFC 7230 Section 6.6)
```

Reply:
I had to adjust the comment, covering both cases (after 31a0fd).